src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/FileTabCharacterCheckTest.java

```diff
@@ -56,9 +58,9 @@ public void testDefault() throws Exception {
             "19:25: " + getCheckMessage(MSG_FILE_CONTAINS_TAB),
         };
         final File[] files = {
-            new File(getPath("InputSimple.java")),
+            new File(getPath("SimpleFileTabCharacterInput.java")),
```

Comment:
Input files should be named after their tests.
`FileTabCharacterCheckTest`'s inputs should be named `InputFileTabCharacter...`.